admin/src/main/java/io/strimzi/arguments/topic/DescribeTopicCommand.java

```diff
@@ -29,13 +37,38 @@ public Integer call() {
      */
     private Integer describeTopics() {
         try (Admin admin = Admin.create(AdminProperties.adminProperties(this.bootstrapServer))) {
-            admin.describeTopics(List.of(this.getTopicPrefixOrName())).topicNameValues().forEach((key, value) -> {
+            List<KafkaTopicDescription> kafkaTopicDescriptionList = new LinkedList<>();
+
+            // parse kafka response
+            admin.describeTopics(this.getListOfTopicNames()).topicNameValues().forEach((key, value) -> {
                 try {
-                    System.out.println(value.get());
+                    final TopicDescription description = value.get();
+                    final int partitionCount = description.partitions().size();
+                    // as each partition has the same size, replica count is deduced from first partition
+                    final int replicaCount = description.partitions().get(0).replicas().size();
+                    KafkaTopicDescription kafkaTopicDescription = new KafkaTopicDescription(key, partitionCount, replicaCount);
+                    kafkaTopicDescriptionList.add(kafkaTopicDescription);
                 } catch (Exception e) {
-                    throw new RuntimeException("Unable to describe topic: " + this.getTopicPrefixOrName() + " due: " + e.getCause());
+                    throw new RuntimeException("Unable to describe topic: " + key + " due: " + e.getCause());
                 }
             });
+
+            // provide admin response
+            switch (outputFormat) {
+                case PLAIN:
+                    for (KafkaTopicDescription topic : kafkaTopicDescriptionList) {
+                        System.out.println("name:" + topic.name() + ", partitions:" + topic.partitionCount() + ", replicas:" + topic.replicaCount());
+                    }
+                    break;
+                case JSON:
+                    ObjectMapper mapper = new ObjectMapper();
+                    System.out.println(mapper.writeValueAsString(kafkaTopicDescriptionList));
+                    break;
+                default:
+                    throw new IllegalArgumentException("Unsupported output format: " + outputFormat);
+            }
```

Comment:
The previous comment applies here as well.
You can have methods like `getPlain()` and `getJson()` (or whatever name) in the `KafkaTopicDescription`
Or add it into some utils class or something.